scripts/update-release-readiness.sh

```
@@ -0,0 +1,284 @@
+declare -A issue_numbers=()
+for pull_number in $(printf '%s\n' "${!pull_numbers[@]}" | sort -n); do
+  closing_issues="$(gh api graphql \
+    -f query='query($owner: String!, $repo: String!, $number: Int!) {
+      repository(owner: $owner, name: $repo) {
+        pullRequest(number: $number) {
+          closingIssuesReferences(first: 100) {
+            nodes { number }
+          }
+        }
+      }
+    }' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)